### ISSUE TITLE
What to send to camera for getdeviceinfo?

### ISSUE THREAD

**Issue by tho068** [felis/Arduino_Camera_Control#3]
Hello.

I know this might not be appropriate since this is not an issue with this implementation or anything. I am just wondering what you actually send to the camera to get the device info, before you enter a session.

I am trying to develop a microchip for controlling the camera, and I've tried to understand your code, to find out how to communicate with the camera.

The USB specs says that the container that is sent to the camera should have:

Container length
Container type
Operation code
Transaction ID
Payload

However, I cant seem to see that you actually includes all of these fields when you ask for device details.

Are not all of them neccessary?

I am sorry if this is not the right place to post, and it probably isnt.

**Comment by felis** [comment]
This could help -> https://www.circuitsathome.com/mcu/programming/digital-camera-control-using-arduino-usb-host-shield-part-1-basics . 

tldr:
No camera data can be obtained before opening a session. From standard USB descriptors you can only learn that the device is indeed a camera.
